### ISSUE TITLE
Samba4 use tls

### ISSUE THREAD

**Issue by marcioducrato** [ltb-project/self-service-password#827]
configuration parameters to use tls in samba4 AD mode

**Comment by coudot** [comment]
Is `$ldap_starttls` not enough?

https://self-service-password.readthedocs.io/en/latest/config_ldap.html

**Comment by marcioducrato** [comment]
no, why no where to define the CA or TLS_REQCERT allow, as samba4 AD does not have the ldap.conf configuration file

**Comment by marcioducrato** [comment]
no, why no where to define the CA or TLS_REQCERT allow, as samba4 AD does not have the ldap.conf configuration file

**Comment by marcioducrato** [comment]
in file conf.inc.php?

**Comment by coudot** [comment]
Please read the doc: https://self-service-password.readthedocs.io/en/latest/config_ldap.html
